### PR TITLE
FX Option notebook for different option products

### PR DESCRIPTION
**Pull request by andrewjlp**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [x] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Details under: https://finbourne.atlassian.net/browse/SE-2489

### CODE REVIEW

File: examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb [hunk 1]
```diff
@@ -0,0 +1,2411 @@
+{
```
Comment (rizwansaeed) [inline examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>remove this output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>Probably need a sentence or 2 explaining what this means, a "normal" user will probably be unaware!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Comment (rizwansaeed) [inline examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # get valuation at expiry - NOTE there is a bug such that the trade can not be valued intrinsically on the expiry date for American and Digital Options (hence using 29 rather 30 days)
</pre><p>should we not get the bug fixed before releasing this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>done</p>

Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>done</p>

[the remark this answers is not in the document]
Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>done</p>

File: examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb [hunk 2]
```diff
@@ -0,0 +1,1415 @@
...
```
Comment (lorenzdec) [inline examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>Can we avoid using -2, _final, and other extensions to scopes used during creation when submitting the final version that we are merging? Can we change this to ibor-option and ibor-opt please.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (lorenzdec) [inline examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>the fxOption instrument into LUSID, not LUCID :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by lorenzdec** [comment]
<p>Great notebook! Clear and concise way to approach a complex topic - just found 1 minor name mistake and would like the scope to be renamed. Apart from that LGTM.</p>

File: examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb (hunk 2 above)
Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>This was done to ensure that the notebook scope is unique, the other option notebooks may clash at build/test time.</p>

Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>Done</p>

Comment (andrewjlp) [reply examples/use-cases/instruments/FX Option - 2 Trade comparison worksheet.ipynb:1]: <p>I have changed to ibor-fxoptions</p>

**Review by lorenzdec — APPROVED** [review]
/LGTM

**Review by rizwansaeed — APPROVED** [review]

**Pull request merged**